fanout/readall_reader.go

```diff
@@ -7,7 +7,7 @@ import (
 	"sync"
 )
 
-//A reader that emits its read to multiple consumers using a ReadAll(p []byte) ([]interface{}, error) func
+// A reader that emits its read to multiple consumers using a ReadAll(p []byte) ([]interface{}, error) func
```

Comment:
Let's add the missing white-space also to the comment in the `toWriters` function in this file.